lab11_4/src/main.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { useState } from 'react';
+
+import './style.css';
+

```

Comment:
The solution is good, however you are basically recreating the native color picker that every browser has in `<input type="color"`

Reply:
thanks